### PR TITLE
Load entity

### PR DESCRIPTION
**Pull request by anncatton**
Initial work for EntityContext: loading a single User entity and displaying meta info in the panel
- Creates custom folder for User content components
- Creates common folder for shared content components
- Moves `EntityContext` and `ListContext` into separate `ContentProvider`, and moved this inside the `ProtectedRoute` in App
- creates main dictionary for panel components
- some file renaming
- display only, no editing state (buttons do not work). Content `mode` is mocked for now
- adds enums for fieldnames
- comments out original content components so compiler doesn't yell

### CODE REVIEW

File: src/components/Content/types.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export enum ContentState {
```
Comment (anncatton) [inline src/components/Content/types.ts:1]: Will be revisiting these, possibly will re-implement some as loading states within the buttons

File: src/components/Content/common/buttons.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    `}
+    // disabled={isSaving}
+    // onClick={() => history.push(`/${resource.name.plural}/create`)}
+  >
+    Create
+  </RippleButton>
+);
+
+export const EditButton = () => (
+  <RippleButton
```
Comment (ciaranschutte) [inline src/components/Content/common/buttons.tsx]: const theme = useTheme()

File: src/components/Content/common/buttons.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    //     setContentState(ContentState.DISPLAYING);
+    //   }
+    // }}
+  >
+    Disable
+  </RippleButton>
+);
+
+export const ConfirmDeleteButton = () => (
+  <RippleButton
```
Comment (ciaranschutte) [inline src/components/Content/common/buttons.tsx]: these buttons are self contained in components.
i think it's cleaner to use the useTheme() hook and just put the color into the css`` call

Comment (ciaranschutte) [reply src/components/Content/common/buttons.tsx]: or even use "styled" like you have used in other places

File: src/components/ListPane/ListPane.tsx [hunk 4]
```diff
@@ -197,12 +197,12 @@ const List = () => {
           })
         }
       /> */}
-      {(listParams.limit < list.count || listParams.offset > 0) && (
+      {(listParams.limit < list?.count || listParams.offset > 0) && (
```
Comment (ciaranschutte) [inline src/components/ListPane/ListPane.tsx]: might be worth just pushing this up to the top level as a `showPagination` var

**Review by ciaranschutte — APPROVED** [review]
lgtm! usual one or two style choices that you can take or leave... just so I'm giving some kind of feedback

File: src/components/Content/common/buttons.tsx (hunk 3 above)
Comment (anncatton) [reply src/components/Content/common/buttons.tsx]: Sweet! That gives me clearer typing too 🙂

**Pull request merged**